### PR TITLE
Multiple-threads loading

### PR DESCRIPTION
**Pull request by VitaZheltyakov**
Multiple-threads loading demo.
This demo shows how you can organize background loading using multiple threads.
Сompatible with the latest stable Godot version (currently 3.2).
For more information, see:
[http://docs.godotengine.org/en/3.2/tutorials/io/background_loading.html#using-multiple-threads](http://docs.godotengine.org/en/3.2/tutorials/io/background_loading.html#using-multiple-threads)

*Bugsquad edit:* Supercedes #281

### CODE REVIEW

File: misc/multiple_threads_loading/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Multiple-threads loading demo
```
Comment (aaronfranke) [inline misc/multiple_threads_loading/README.md]: I just recently added README files to the other demos. Please reformat the README to be similar to the others. In particular, remove the sections about Godot versions (not useful documentation) and License (not necessary, the repo as a whole already has a license, the only time it's necessary to specify a license is if specific files are under a different one).

File: misc/multiple_threads_loading/preload.gd [hunk 2]
```diff
@@ -0,0 +1,27 @@
+extends Node
+
+var queue
+
+func _ready():
+	queue = preload("res://resource_queue.gd").new()
+	# Call after you instance the class to start the thread.
+	queue.start()
+
+func _process(delta):
```
Comment (aaronfranke) [inline misc/multiple_threads_loading/preload.gd]: The warnings need to be silenced. `delta` is unused, so prefix it with an underscore: `_delta`.

Please be sure to follow the [GDScript style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html). In particular, use two lines between functions.

**Comment by VitaZheltyakov** [comment]
@aaronfranke I adjusted the request as you wanted (I hope).

I want to tell you that your behavior is very disgusting.
You want developers to adapt to your needs. I'm not a telepath. I can 't know what you want to see in the readme. 
If you really wanted to help, you would have accepted the request and made the changes yourself.

**Comment by aaronfranke** [comment]
There is now a "loading" folder for loading demos. This needs to be moved there.

**Comment by VitaZheltyakov** [comment]
@aaronfranke I made a change.
As I understand it , you are guided by some rules. Where can I find these rules?
You should have given me a link to them right away. Since I spend a lot of time trying to guess your queries.

**Comment by aaronfranke** [comment]
The rules are:

* The demo should work. This includes no errors or warnings.

* Follow the official style guides, such as the [GDScript style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html).

* Format your demo similar to other demos. This can change over time, especially recently. For years the demo projects had no maintainer, I was given write access a bit over a month ago and I've been working on improving it in my spare time between college classes. Please understand that we don't currently have a formal guide for this, but there are discussions [here](https://github.com/godotengine/godot-docs/issues/3104) and [here](https://github.com/godotengine/godot-demo-projects/issues/390) and [here](https://github.com/godotengine/godot-demo-projects/issues/349).

Also, please understand that once a demo is in this repo, the Godot team has a responsibility to maintain it. We want to make the demos as good as possible the first time they're added rather than accepting code and changing it later.

TL;DR: Make it work, follow the style guides, sorry that we don't have guides for everything, I'm an unpaid maintainer working on improving this situation in my spare time.

**Comment by VitaZheltyakov** [comment]
@aaronfranke Your actions only interfere with development.
I and other developers have to guess what you want from us. This is offensive. 
If you want to help, write a list of rules and approve it. And then encourage the enforcement of these rules.

**Comment by aaronfranke** [comment]
Please read the [code of conduct](https://godotengine.org/code-of-conduct), in particular:

> Politeness is expected at all times. Be kind and courteous.
> Always assume positive intent from others.

I will write a comprehensive list of rules eventually, but we don't have such a thing right now. In the meantime, I've provided you with what I can.

**Comment by VitaZheltyakov** [comment]
I understand what you want from me (I hope). 

@aaronfranke You create such negative situations yourself.
You force developers to look for rules for writing readmis; to guess your wishes. 
If you had written at once that it was a demonstration, this situation would not have happened . You wrote "this". What does "this" mean ?
